src/v2/Components/v2/ArtworkFilter/ArtworkFilters/SizeFilter2.tsx

```diff
@@ -172,18 +176,16 @@ export const SizeFilter2: React.FC<SizeFilter2Props> = ({ expanded }) => {
               name="width_min"
               min="0"
               step="1"
-              value={customSize.width && customSize.width[0]}
```

Comment:
These conditional checks don't appear to be needed; there's always a value there.

Reply:
Actually I believe if you only set one of the dimensions (so come to the page with a URL like `?height=12-24` there'd be an issue).

Reply:
Should be good to go now.